chainer/optimizer.py

```diff
@@ -652,9 +655,58 @@ def serialize(self, serializer):
             if rule is not None:
                 rule.serialize(serializer[name])
 
+    def loss_scaling(self, interval=1000, scale=None):
+        """Sets loss scaling configurations.
+
+        Args:
+            interval (integer): Number of iterations until scaling factor gets
+                doubled. This is effective when "dynamic" loss scaling is used.
+            scale (float): Loss scaling factor. If None, "dynamic" loss scaling
```

Comment:
A minor but how about the following (I'm not sure if this exceeds the maximum line length)?
```suggestion
            scale (float): Loss scaling factor. If ``None``, "dynamic" loss scaling
```